src/radar.cpp

```diff
@@ -184,7 +184,6 @@ bool resizeRadar()
 	debug(LOG_WZ, "Setting radar zoom to %u", RadarZoom);
 	radarSize(RadarZoom);
 	pie_SetRadar(-static_cast<float>(radarWidth) / 2.0 - 1, -static_cast<float>(radarHeight) / 2.0 - 1, static_cast<float>(radarWidth), static_cast<float>(radarHeight), radarTexWidth, radarTexHeight);
-	setViewingWindow();
```

Comment:
This caused an assert failure due to uploading buffers multiple times during the same frame. Removing the call seems to have fixed the issue and I have not noticed anything breaking due to this.